presto-native-execution/presto_cpp/main/Announcer.h

```diff
@@ -34,6 +34,7 @@ class Announcer : public PeriodicServiceInventoryManager {
       const bool sidecar,
       const std::vector<std::string>& connectorIds,
       const uint64_t maxFrequencyMs_,
+      const std::string& nodePoolType,
```

Comment:
We could move this parameter higher up since there aren't any optional parameters in this method. After nodeLocation or sidecar ?